lib/checkPullRequestReview.js

```diff
@@ -16,6 +16,7 @@
  * @fileoverview File to handle checks when a PR gets reviewed.
  */
 const utilityModule = require('./utils');
+const { pingAndAssignReviewers } = require('./utils');
```

Comment:
You can directly use utilityModule. pingAndAssignReviewers. It is already imported.